draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,88 @@ Appraisal Policy to make application-specific decisions such as authorization de
 The Attestation Result Appraisal Policy might, for example, be configured in the Relying Party 
 by an administrator.
 
+## Composite Attester {#compositeattester}
+
+A Composite Attester is an entity composed of multiple sub-entities that its
+trustworthiness has to be determined by evaluating all these sub-entities.
+Each sub-entity has at least one Attesting Environment collecting the claims 
+from at least one Target Environment, then this sub-entity generates Evidence
+about its trustworthiness. Therefore each sub-entity can be called an Attester.
+Among these Attesters, there may be only some, which can be called Lead Attesters,
+that have the communication functionality with the Verifier. Other Attesters
+don't have this ability, but they are connected to the Lead Attesters via internal
+links or network connections, and they are evaluated via the Lead Attesters' help.
+
+For example, the carrier-grade router is a composite device consisting of a chassis and multiple slots.
+The trustworthiness of the router depends on all its slots' trustworthiness.
+Each slot has the attesting environment like the TPM or TEE collecting the
+claims of its boot process and generates Evidence to prove its trustworthiness.
+Among these slots, only the main control slot can communicate with the Verifier
```

Comment:
What's a "control" slot?  The term "control" doesn't appear next to "slot" anywhere above.

Reply:
I think this is a translation problem, I've deleted the word "control". "main slot" is enough to show the meaning.